mlir/include/mlir/Dialect/ArmSME/Transforms/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ set(LLVM_TARGET_DEFINITIONS Passes.td)
 mlir_tablegen(Passes.h.inc -gen-pass-decls -name ArmSME)
 mlir_tablegen(PassesEnums.h.inc -gen-enum-decls)
 mlir_tablegen(PassesEnums.cpp.inc -gen-enum-defs)
+mlir_tablegen(Passes.capi.h.inc -gen-pass-capi-header --prefix ArmSME)
+mlir_tablegen(Passes.capi.cpp.inc -gen-pass-capi-impl --prefix ArmSME)
```

Comment:
Should I also include the `PassesEnums` into the C API?

Reply:
Ah yes good point, could be useful too.
There may also be some non-pass enums missing, while you're at it.

Reply:
Can you point me which enums are those? I'm not familiar with the LLVM codebase.

Reply:
my apologies, my dated recollections point me to python bindings specifics that may be specific to Google's bazel stuff: https://github.com/llvm/llvm-project/blob/main/utils/bazel/llvm-project-overlay/mlir/python/BUILD.bazel#L195

I thin you can safely ignore